packages/account/src/Components/personal-details/__tests__/personal-details.spec.tsx

```diff
@@ -10,6 +11,11 @@ import { StoreProvider, mockStore } from '@deriv/stores';
 import { Analytics } from '@deriv-com/analytics';
 import { FormikErrors } from 'formik';
 
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({ isDesktop: true, isTablet: false, isMobile: false })),
```

Comment:
```suggestion
    useDevice: jest.fn(() => ({ isDesktop: true })),
```
the defaults are `false`, right?

Reply:
@maryia-deriv the defaults are `undefined`, because I mock the object which will be returned from this hook. With your suggestion `isMobile` and `isTablet` will be `undefined`